### ISSUE TITLE
Low FPS on Start up for extended period of time

### ISSUE THREAD

**Issue by Duplo52** [MJRLegends/Space-Astronomy-2-Feedback-#38]
### ---Issue Report---

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** [Yes or No]
 Yes, 1 that was not solved due to lack of info and 1 that was on a much older version.

### Description of Issue
[Description of the issue]
I load a new world and wait for the world to generate for a few min. Kept waiting for about 15+min to see if lag subsides but stays strong at 1-3FPS.

### Steps to Reproduce Issue
1. Open pack
2. Generate new world, Wait.

**Crash log/Log (if applicable):** [PASTEBIN. Do NOT copy and paste the log into the issue post itself.] No crash log as the game doesn't crash per say, and the regular log is to large for a pastebin. If there is a different way to get you the regular log I can get that to you also.

**Version of Mod Pack using:** [VERSION]  
1.1.3 beta

### Additional Information
[Any other information that may be able to help me with the problem]
Screen shots http://imgur.com/a/oOpEo

**Comment by Maudslay** [comment]
For one, An I5-3350P along with AMD Radeon HD 7800 Graphics, is the main cause of this lag as your Pc or Laptop by the looks of it can't handle the pack mate, I have a pretty strong PC and The pack is very heavy on CPU side as it's a 1.10.2 based Pack, The newer version of Minecraft has updated models that require higher amounts of CPU and RAM and VRAM to manage the Game. So therefore you might need to upgrade your Equipment to continue playing the pack mate. This is my opinion but you know :)

**Comment by Duplo52** [comment]
Thanks for the reply. It is a desktop actually. I can run the other 1.10.2 packs just fine. I understand this is a heavier pack but could it be THAT heavy that its in a class of its own compared to the rest? This is the first one I've had this issue with, the lag dissapates in game when have the esc. menu open. is there a reason my fps would be fine with that open and not when playing?

**Comment by Maudslay** [comment]
Try install Optifine because i use it and it works perfectly :) i hope i helped

**Comment by Duplo52** [comment]
Thanks Maudslay. I had seen a post from MJR about Optifine not being supported by mod devs so I was trying to find another solution however I did install it and go through the animations in the video settings turning them all off worked. So I troubleshot through the list to discover the "Terrain Animated" option (http://imgur.com/a/Lbzym) was the cause. Now I can't tell what in particular it will have a conflict with being off but I can play the pack now.  One could say the problem was solved with optifine options ONLY.
